### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 10

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#476] [Esami & Esoneri M-Z]
### Traccia

<img width="925" alt="image" src="https://github.com/user-attachments/assets/7508087e-067b-4f31-8b5d-51ad12da46d4">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/6b1a493d-ac8e-418c-9fce-076b758134f1)
